docs/profile/profile.rst

```diff
@@ -624,6 +624,117 @@ Example
 
 
 
+==============================================================================
+
+.. _profile_setIdentityAccess:
+
+setIdentityAccess
+================================================================================
+
+.. code-block:: typescript
+
+  profile.setIdentityAccess(address, key);
+
+Function description
+
+Add identity key to address book
+
+----------
+Parameters
+----------
+
+#. ``address`` - ``string``: identity address
+#. ``key`` - ``string``: identity datakey
+
+-------
+Returns
+-------
+
+``Promise`` returns ``void``: resolved when done
+
+-------
+Example
+-------
+
+.. code-block:: typescript
+
+  await profile.setIdentityAccess(identities[0], 'key 0x01_b');
+
+
+
+==============================================================================
+
+.. _profile_removeIdentityAccess:
+
+removeIdentityAccess
+================================================================================
+
+.. code-block:: typescript
+
+  profile.removeIdentityAccess(address);
+
+Function description
+
+Remove identity key from address book
+
+----------
+Parameters
+----------
+
+#. ``address`` - ``string``: identity address to be removed
+
+-------
+Returns
+-------
+
+``Promise`` returns ``void``: resolved when done
+
+-------
+Example
+-------
+
+.. code-block:: typescript
+
+  await profile.removeIdentityAccess(identities[0]);
+
+
+==============================================================================
+
+.. _profile_getIdentityAccessList:
+
+getIdentityAccessList
+================================================================================
+
+.. code-block:: typescript
+
+  profile.getIdentityAccessList();
+
+Function description
+
+Gets a list of identities, where the profile identity has access to, from address book. Keep in mind address book needs to be loaded beforehand using the :ref:`loadForAccount` API.
```

Comment:
Can you add some more description or a more extended example about the result here? Something like, this will return an object, with identity addresses mapped to it's encryption keys and sha3 addresses mapped to their identity address.